### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by BrunCola**
- manifed.json -> "matches": ["<all_urls>"], added that because the extension was not picking up all URLs

- popup.js -> added some styling (with the styles.css) changed it to a toggle instead of a checkbox
-> also added some css for when a feature title is very long to add "..."

- state.js -> added some logic, for everytime you change tag, to use the tabId (instead of the tab object)

- plugin.js -> added a check to only parse on string (broke when it was null)
-> also added the sender parameter in the chrome.runtime.onMessage.addListener function .. (it did not have the correct number of params)

### CODE REVIEW

File: src/popup.js [hunk 1]
```diff
@@ -9,27 +9,43 @@ function render() {
...
-            });
-            const label = document.createElement('label');
-            label.setAttribute('for', feature);
-            label.innerText = features[feature].name;
-            wrapper.appendChild(el);
-            wrapper.appendChild(label);
-
-            container.appendChild(wrapper);
+
+                let label = document.createElement('label');
```
Comment (phroggyy) [inline src/popup.js]: You should be able to keep this `const`

File: src/popup.js [hunk 2]
```diff
@@ -9,27 +9,43 @@ function render() {
...
+                label.setAttribute('for', feature);
+                label.setAttribute('class', 'switch');
+
+                let toggleText = document.createElement('div');
```
Comment (phroggyy) [inline src/popup.js]: Same here

File: src/popup.js [hunk 3]
```diff
@@ -9,27 +9,43 @@ function render() {
...
+                toggleText.innerText = features[feature].name;
+                toggleText.setAttribute('class', 'switchText');
+
+                let span = document.createElement('span');
```
Comment (phroggyy) [inline src/popup.js]: And here

File: src/popup.js [hunk 4]
```diff
@@ -9,27 +9,43 @@ function render() {
...
-            el.setAttribute('id', feature);
-            el.setAttribute('checked', `${features[feature].enabled}`);
-            el.addEventListener('change', function (event) {
-                features[event.target.id].enabled = event.target.checked;
-
-                chrome.tabs.query({ active: true, currentWindow: true }, function (tabs) {
-                    chrome.tabs.sendMessage(tabs[0].id, {event: 'featureToggled', 'feature': event.target.id, value: features[event.target.id] });
+            for (const feature of Object.keys(features)) {
+                let wrapper = document.createElement('div');
+                let el = document.createElement('input');
```
Comment (phroggyy) [inline src/popup.js]: These could remain `const`

**Review by phroggyy — APPROVED** [review]

**Comment by phroggyy** [comment]
Thanks for the awesome work! Had a couple small comments on using `const` over `let` but it looks good overall!

**Comment by appelsiini** [comment]
Excellent stuff guys! Great to see immediate participation here, taking in account the project is brand new. 😎👌

File: src/popup.js (hunk 4 above)
Comment (BrunCola) [reply src/popup.js]: thanks for the advice!

**Pull request merged**